mods/ts/rules/shared-vehicles.yaml

```diff
@@ -129,7 +129,7 @@ LPST:
 	Mobile:
 		Speed: 85
 		TurnSpeed: 5
-		RequiresCondition: !empdisable && undeployed
+		RequiresCondition: !empdisable && ?undeployed
```

Comment:
I get the impression that this `?` operator and replacing `||` with `+` have come from my request to force like-types around binary operators.  If this is indeed the case, then we should drop that requirement.